.NET/Microsoft.Recognizers.Definitions.Common/English/TimeZoneDefinitions.cs

```diff
@@ -20,6 +20,299 @@ public static class TimeZoneDefinitions
+		public static readonly IList<string> AbbreviationsList = new List<string>
+		{
+			@"AFT",
```

Comment:
No need to duplicate regex and list. You can pre-process the regexes to build these lists.